mk/spksrc.python-wheel.mk

```diff
@@ -25,7 +25,13 @@ PYTHONPATH = $(PYTHON_LIB_NATIVE):$(INSTALL_DIR)$(INSTALL_PREFIX)/$(PYTHON_LIB_D
 
 ### Python wheel rules
 build_python_wheel:
+ifeq ($(strip $(CROSSENV)),)
```

Comment:
If `CROSSENV` environment variable isn't defined we'll assume we're going down the Python 2 road.
Otherwise we "source" the `$(CROSSENV)` script before running the build command using the `python` (aliased by the `$(CROSSENV)` script